scripts/control-plane-dpu/on-server/post-power-cycle.sh

```
@@ -0,0 +1,83 @@
+
+# shellcheck source=dpuinstall.sh
+source "$WORK_DIR/dpuinstall.sh"
+
+[[ -f "$TOUCHFILE_HBN_DEPLOYED" ]] || die "HBN deployment touchfile not found — power cycle may have happened too early or install-dpu.sh did not complete"
+
+trap cleanup EXIT
+set -eux
+
+update_progress 11
+check_hbn_container
+CUR_STEP=$FINAL_STEP
```

Shellcheck (0.11.0)

[warning] CUR_STEP appears unused. Verify use (or export if used externally).

(SC2034)